src/file.ts

```diff
@@ -326,6 +325,24 @@ export const STORAGE_POST_POLICY_BASE_URL = 'https://storage.googleapis.com';
  */
 const GS_URL_REGEXP = /^gs:\/\/([a-z0-9_.-]+)\/(.+)$/;
 
+/**
+ * @private
```

Comment:
Perhaps a comment or [@privateRemarks](https://tsdoc.org/pages/tags/privateremarks/) on how to regenerate this regex over time would keep this maintainable in the future